charts/spire/charts/spire-server/values.yaml

```diff
@@ -73,7 +73,8 @@ affinity: {}
 
 topologySpreadConstraints: []
 
-dataStorage:
+# Used by sqlite3 for spire-server, by the disk KeyStore, and/or by Tornjak.
```

Comment:
This is a bit confusing. and/or, if it is a or, it should probably have dedicated PVC so you can pick the desired storage class and capacity for that specific need.

To have it show up in the docs please use the helm-docs syntax.

Reply:
'and/or' is kind of a weird English thing because its imprecise and there is no word for the possibility that between 1 and all of something is possible. I'm open to a different wording though. Wording is hard.

Dedicated PVC's for each task would be quite wasteful. I've experienced bugs with pvcs that were allocated < 1gb. The disk KeyStore needs to persistently store kb of data. Tornjak and spire-server sqlite will likely be in the 10's of mb. So its already pretty wasteful at a minimal 1gb and it would be up to 3x wasteful having one per.

I'll update it to be helm-docs format.

Reply:
Updated